Makefile

```diff
@@ -1,5 +1,5 @@
-MDBOOK_VERSION ?= 0.4.5
-MDBOOK_BIN_VERSION ?= v0.4.5
```

Comment:
I couldn't find these tags here: https://quay.io/repository/metal3-io/mdbook?tab=tags

Reply:
Those are not quay image tags, but should be mdbook release version tag itself (https://github.com/rust-lang/mdBook/tags)

Reply:
Ok! I've reverted this. I think we should push a tagged mdbook image to Quay though, because IMAGE_NAME points at quay, and this tag is missing.